src/basic/terminal-util.c

```diff
@@ -589,10 +589,21 @@ int make_console_stdio(void) {
                         return log_error_errno(r, "Failed to make /dev/null stdin/stdout/stderr: %m");
 
         } else {
+                unsigned rows, cols;
+
                 r = reset_terminal_fd(fd, true);
```

Comment:
If you touch the code, then please add a short comment about the boolean argument.